### PR TITLE
Updated UI of Contact Form

### LINKED ISSUE #247: Improve UI of Contact Form in Basic Web Development Section #GSSoc 22
Improving the UI :  
 • adding 'glassmorphism' web design to the form
 • application of vanilla
 • adding a favicon                          

Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?
Yes , I have go through the contributing.md file .

## Motivation
UI/UX design of Web Apps has always been my domain of interest. My aim is to keep getting better at this.

## Pitch
Improving the UI will lead to enhanced user experience.
#GSSoc 22

### PR DESCRIPTION
**Pull request by shania-balkhi**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes, I have read it thoroughly.

### Description

• added 'glassmorphism' web design to the form.
• applied vanilla-tilt effect to the form to make the form visually attractive and playful.
• added the favicon of GSSoC so that the page appears authentic, original and complete. 
• the site now seems visually attractive and user friendly!

Fixes #247 

### Checklist
- [✓ ] I've read the contribution guidelines.
- [ ✓] I've checked the issue list before deciding what to submit.
- [✓ ] I've edited the `README.md` and link to my code.

### CODE REVIEW

File: Web Development/Basic/Contact Form/README.md [hunk 1]
```diff
@@ -18,6 +18,5 @@ if(Love the repo)
 Show your 💖 by Staring Some of My Repo and by following🙂 me.
 }
 ```
-## [Try Now](https://ug-sep.github.io/Contact-Form/)☺
+## [Try Now](https://shania-balkhi.github.io/Project-Guidance/)☺
```
Comment (Kushal997-das) [inline Web Development/Basic/Contact Form/README.md:21]: Delete this !!

File: _config.yml [hunk 2]
```diff
@@ -1 +1 @@
-theme: jekyll-theme-dinky
+theme: jekyll-theme-cayman
```
Comment (Kushal997-das) [inline _config.yml:1]: Delete this one !

File: Web Development/Basic/Contact Form/README.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
```
Comment (Kushal997-das) [inline Web Development/Basic/Contact Form/README.md:2]: Don't leave white space here !

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@shania-balkhi Changes all the things which i mentioned and delete this (Web Development/Basic/Contact Form/favicon.png)file.

**Comment by Kushal997-das** [comment]
@shania-balkhi OMG You added duplicated `webdevelopment` folder please delete this

File: _config.yml (hunk 2 above)
Comment (shania-balkhi) [reply _config.yml:1]: Done Sir!

File: Web Development/Basic/Contact Form/README.md (hunk 3 above)
Comment (shania-balkhi) [reply Web Development/Basic/Contact Form/README.md:2]: Done Sir!

**Comment by shania-balkhi** [comment]
> @shania-balkhi OMG You added duplicated `webdevelopment` folder please delete this

Okay sir!

**Comment by Kushal997-das** [comment]
@shania-balkhi ?

**Comment by Kushal997-das** [comment]
@shania-balkhi Are you going to do the changes ? Or else your PR will close .

**Comment by shania-balkhi** [comment]
> @shania-balkhi Are you going to do the changes ? Or else your PR will close .

@Kushal997-das I'm on it. By the end of this week it'll be done. (had gotten stuck because of the re-opening of my college). Kindly wait till I work on the task!

**Pull request closed without merging**